ScriptingExample/src/test/java/org/openzen/zenscript/scriptingexample/tests/actual_test/java_native/enums/EnumMemberTests.java

```diff
@@ -28,6 +28,34 @@ void EnumMembersMustBeRetrievableByTypeHintAndName() {
 		logger.assertPrintOutput(0, "A");
 	}
 
+	@Test
+	void EnumMembersMustBeRetrievableByFullName() {
```

Comment:
Moved these from TypeInferenceBlockingTests since honestly I have no idea why we named that other class that way 😅